### PR TITLE
[VITIS-5805] XRT Linux: Remove legacy qdma driver support

### PR DESCRIPTION
**Pull request by rajkumar-xilinx**
Remove legacy qdma driver which is no longer needed in XRT. Plan is to add new qdma driver support on versal platforms.

Signed-off-by: Rajkumar Rampelli <rajkumar@xilinx.com>

#### Problem solved by the commit
Remove legacy qdma driver which is no longer needed.
#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
VITIS-5805 legacy qdma driver is no longer needed
#### How problem was solved, alternative solutions (if any) and why they were rejected
Removed qdma driver from XRT
#### Risks (if any) associated the changes in the commit
NA
#### What has been tested and how, request additional testing if necessary
xbutil/xbmgmt tool tests
#### Documentation impact (if any)
NA

### CODE REVIEW

**Comment by rajkumar-xilinx** [comment]
As XRT no longer need old qdma drivers, removing it and the same is confirmed with @sonals .
Plan is to support latest qdma driver (cpm4) on versal platforms. Work is in progress to get these drivers into XRT using "git submodule".

**Review by chvamshi-xilinx — APPROVED** [review]

**Review by houlz0507 — COMMENTED** [review]
Could you search NODE_QDMA and cleanup: devices.h, xocl_fdt.c, xocl_fdt.h ?

**Comment by rajkumar-xilinx** [comment]
> Could you search NODE_QDMA and cleanup: devices.h, xocl_fdt.c, xocl_fdt.h ?

Hi @houlz0507 , Since the endpoint name is ep_qdma_00, the new qdma driver will make use of these nodes and relevant code in device/xocl_fdt.* files. So, I have not removed this code for now. Thank you

**Comment by rajkumar-xilinx** [comment]
Hi all, could you please review this PR, thank you.

**Comment by houlz0507** [comment]
> > Could you search NODE_QDMA and cleanup: devices.h, xocl_fdt.c, xocl_fdt.h ?
> 
> Hi @houlz0507 , Since the endpoint name is ep_qdma_00, the new qdma driver will make use of these nodes and relevant code in device/xocl_fdt.* files. So, I have not removed this code for now. Thank you

ok. What about devices.h? there are some legacy qdma platforms need to be removed. e.g. MGMT_XBB_QDMA_U280

**Review by maxzhen — APPROVED** [review]

**Comment by rajkumar-xilinx** [comment]
Hi @houlz0507 , I have updated device.h file by removing QDMA unused entries, please review it. 
I will clean this devices.h file further while integrating the latest QDMA drivers.

**Review by houlz0507 — APPROVED** [review]

File: src/runtime_src/core/pcie/driver/linux/xocl/lib/Makefile.in [hunk 1]
```diff
@@ -15,24 +15,6 @@
 ccflags-y	+= -D_QDMA4_DEBUGFS_
```
Comment (rozumx) [inline src/runtime_src/core/pcie/driver/linux/xocl/lib/Makefile.in:15]: Please update the copyright date - Line 2

**Review by rozumx — APPROVED** [review]

**Review by stsoe — APPROVED** [review]

**Pull request merged**